query-connector/src/app/shared/query-service.ts

```diff
@@ -317,8 +316,8 @@ class QueryService {
         fetchString += `?${k}=${v}`;
       });
       response = await fhirClient.get(fetchString);
-      queryResponse = await parseFhirSearch(response, queryResponse);
-      return queryResponse;
+      this.runningQueryResponse = await this.parseFhirSearch(response);
```

Comment:
needed to do this since passing around this piece of query state as a param was resulting in PHI getting added to the audit log. Since we're making classes for these query services, elected to bundle the running response into a class property since we have ready access to that already